DESCRIPTION.md

```diff
@@ -1,4 +1,6 @@
-My App
+Description
 ======
 
-todo: add a description that will appear alongside your app in Deskpro admin
+***Link devices to users to tickets:*** Linking devices to users in the app lets agents easily view information from Lansweeper about your user's device while interacting with them, making communication more efficient.
```

Comment:
Not sure this is correct, "Link devices to users to tickets". Is that meant to be something like "Link devices to users and/or tickets"

Reply:
Everything is correct here. This application works in the user sidebar. Therefore, devices can only be linked to the user.